drivers/modem/gsm_ppp.c

```diff
@@ -30,6 +31,15 @@ LOG_MODULE_REGISTER(modem_gsm, CONFIG_MODEM_LOG_LEVEL);
 #define GSM_RECV_BUF_SIZE               128
 #define GSM_ATTACH_RETRY_DELAY_MSEC     1000
 
+/* Modem network registration state */
+enum network_state {
+	NET_NOT_REGISTERED = 0,
```

Comment:
These names might conflict with other network related symbols because of the same `NET_` prefix. Could you have here `GSM_` or `GSM_NET_` instead?

Reply:
Sure thing, I will change them to `GSM_*` tomorrow